etc/config/ada.amazon.properties

```diff
@@ -37,33 +37,90 @@ group.gnatriscv64.groupName=GNAT riscv64
 group.gnatriscv64.instructionSet=riscv
 group.gnatriscv64.baseName=riscv64 gnat
 group.gnatriscv64.isSemVer=true
+group.gnatriscv64.supportsBinary=false
+group.gnatriscv64.supportsExecute=false
 
 compiler.gnatriscv64103.exe=/opt/compiler-explorer/riscv64/gnat-riscv64-elf-linux64-10.3.0-2/bin/riscv64-elf-gnat
+compiler.gnatriscv64103.name=riscv64 gnat 10.3.0-2 (Alire)
 compiler.gnatriscv64103.semver=10.3.0
-compiler.gnatriscv64103.supportsBinary=false
 compiler.gnatriscv64103.adarts=/opt/compiler-explorer/riscv64/gnat-riscv64-elf-linux64-10.3.0-2/riscv64-elf/lib/gnat/zfp-rv64imac
 
 compiler.gnatriscv64112.exe=/opt/compiler-explorer/riscv64/gnat-riscv64-elf-linux64-11.2.0-3/bin/riscv64-elf-gnat
+compiler.gnatriscv64103.name=riscv64 gnat 11.2.0-3 (Alire)
 compiler.gnatriscv64112.semver=11.2.0
-compiler.gnatriscv64112.supportsBinary=false
 compiler.gnatriscv64112.adarts=/opt/compiler-explorer/riscv64/gnat-riscv64-elf-linux64-11.2.0-3/riscv64-elf/lib/gnat/zfp-rv64imac
 
+################################
+# GNAT for s390x
+group.gnats390x.compilers=gnats390x1120
+group.gnats390x.groupName=GNAT s390x
+group.gnats390x.instructionSet=s390x
+group.gnats390x.baseName=s390x gnat
+group.gnats390x.isSemVer=true
+group.gnats390x.supportsBinary=false
+group.gnats390x.supportsExecute=false
+
+compiler.gnats390x1120.exe=/opt/compiler-explorer/s390x/gcc-11.2.0/s390x-ibm-linux-gnu/bin/s390x-ibm-linux-gnu-gnat
+compiler.gnats390x1120.semver=11.2.0
+compiler.gnats390x1120.name=s390x GNAT 11.2.0
```

Comment:
Just a heads-up for next time: If you don't provide a name, the compiler name will be set to `${baseName} ${semver}` which seems like it would work in here and in some other places, right?

Reply:
I thought there was something like this but didn't take the time to really look into it. Thanks for the info! I'll rework the patch one more time trying to use this mechanism !